lib/WP_Auth0_LoginManager.php

```diff
@@ -51,49 +33,10 @@ class WP_Auth0_LoginManager {
 	 *
 	 * @param WP_Auth0_UsersRepo    $users_repo - see member variable doc comment.
 	 * @param WP_Auth0_Options|null $a0_options - see member variable doc comment.
-	 * @param null|bool             $admin_role - @deprecated - 3.8.0.
-	 * @param bool                  $ignore_unverified_email - @deprecated - 3.8.0.
 	 */
-	public function __construct(
-		WP_Auth0_UsersRepo $users_repo,
-		$a0_options = null,
-		$admin_role = null,
-		$ignore_unverified_email = false
-	) {
+	public function __construct( WP_Auth0_UsersRepo $users_repo, WP_Auth0_Options $a0_options ) {
```

Comment:
If I was calling this previously passing 3 or 4 parameters, isn't this now a breaking change?

Reply:
Ah sorry. Again I realized after submitting the review that the milestone is `4.0.0`. What is confusing is that you are making all these PRs against master.

Reply:
It's rare that people install plugins into WordPress from a repo and there is a note on the README.

Reply:
But ... just because I'm here ... additional parameters passed to a function/method in PHP do not cause an error of any kind.